changelogs/fragments/suders-user-group-xor.yaml

```diff
@@ -0,0 +1,3 @@
+bugfixes:
+  - |
+    Suders module needs to have either user, or group, set.
```

Comment:
```suggestion
    sudoers - one of the ``user`` and ``group`` options must be supplied. In case none was supplied, the module crashed (https://github.com/ansible-collections/community.general/pull/7823).
```